cleanlab/filter.py

```diff
@@ -493,7 +494,10 @@ def _find_label_issues_multilabel(
     This is done via a one-vs-rest reduction for each class and the results are subsequently aggregated across all classes.
     Here `labels` must be formatted as an iterable of iterables, e.g. ``List[List[int]]``.
     """
-
+    warnings.warn(
+        "The '_find_label_issues_multilabel' function is deprecated and will be removed in future versions. Please use the Cleanlab.multilabel_classification.filter instead.",
```

Comment:
This warning is supposed to appear if the user calls:

```
filter.find_label_issues(..., multi_label=True)
```

and should say: 

The `multi_label` argument to `filter.find_label_issues()` is deprecated and will be removed in future versions. Please use `multilabel_classification.filter.find-label_issues()` instead.

Note that both methods can still just internally call: `_find_label_issues_multilabel` that is no problem

Reply:
Changed here: https://github.com/cleanlab/cleanlab/blob/361e05284c5bb5151a49aee7e5aa8ff41ad47220/cleanlab/filter.py#L316